contents/docs/self-host/configure/migrate-postgres-clickhouse.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Migrate from Postgres to ClickHouse
+sidebar: Docs
+showTitle: true
+---
+
+> This documentation is in **Alpha**. Please provide feedback via the [PostHog Communty Slack workspace](/slack) or [via a GitHub issue](https://github.com/PostHog/posthog.com/issues).
+
+**PostHog backed by Postgres is now deprecated**. We will continue to provide support to Postgres backed installs but we strongly encourage you to migrate to PostHog backed by ClickHouse for improved performance, to receive new features, and continued support in the future.
+
+TODOs:
+- Confirm what will and will not be migrated
+
+## What will be migrated
+
+The following will be migrated:
+
+- Insights
+- Dashboards
+- Sessions
+- Annotations
+- Feature flags
+- Projects
+
+## What will not be migrated
+
+- Cohorts
```

Comment:
@yakkomajuri can you confirm what will and what will not be migrated as part of this process?

For example, should PostHog app users be migrated?

Any other logical entities that should be listed in **will** and **will not**, above?